README.md

```diff
@@ -31,21 +31,25 @@ Additionally, it also includes various deployment scripts that are integral for
+| [`L1VestingWallet`](src/L1/L1VestingWallet.sol) | Vesting wallet for LSK tokens on Ethereum L1 network |
 
 ### Contracts deployed to L2
 
-| Name                                                | Description                                                                                                                                    |
-| --------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------- |
-| [`L2LiskToken`](src/L2/L2LiskToken.sol)             | Bridged Lisk token (LSK) deployed on Lisk L2 network                                                                                           |
-| [`L2Claim`](src/L2/L2Claim.sol)                     | Smart contract responsible for a claiming process of the LSK tokens                                                                            |
-| [`L2Governor`](src/L2/L2Governor.sol)               | The [Governor](https://docs.openzeppelin.com/contracts/5.x/governance) contract for the Lisk DAO which manages proposals                       |
-| [`L2VotingPower`](src/L2/L2VotingPower.sol)         | The [token](https://docs.openzeppelin.com/contracts/5.x/governance#token) contract for the Lisk DAO which manages the voting power of accounts |
-| [`L2Staking`](src/L2/L2Staking.sol)                 | Responsible for locking, unlocking and editing locking positions                                                                               |
-| [`L2LockingPosition`](src/L2/L2LockingPosition.sol) | Stores all information of locking positions                                                                                                    |
-| [`L2Reward`](src/L2/L2Reward.sol)                   | Responsible for staking rewards                                                                                                                |
+| Name                                                                                                                                                                       | Description                                                                                                                                    |
+| -------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------- |
+| [`L2Airdrop`](src/L2/L2Airdrop.sol)                                                                                                                                        | Smart contract responsible for the Lisk Migration Airdrop                                                                                      |
```

Comment:
We have other smart contracts listed here before their deployments.
I just didn't list Airdrop contract inside `Deployments` section because it was not deployed yet.

If you prefer to also remove it from `Contracts Overview` I may do that.

Reply:
ok, not a big deal. I got confused by the header "Contracts deployed to L2" as it suggests that these contracts were already deployed. We could
- reformulate "Contracts deployed to L2" to "Contracts for L2" or
- just leave it as it is